Parsers/ASimAuditEvent/Parsers/ASimAuditEventVectraCloudAccountDetection.yaml

```diff
@@ -0,0 +1,57 @@
+Parser:
```

Comment:
Hi @devikamehra , 

We mapped the detection data with the AuditEvent Schema as that's the closest match we found. Though the detection data is not Audit data, it will be continuously updating based on the detection modification. 

If not AuditEvent Schema, what schema would you suggest us to map detection data with?

Reply:
@dhruvilbhatt-crest Currently we do not have any schema to map these detections. For alerts, we usually create Alert/Incidents using Analytical Rules and thus SecurityAlert table is the source of truth.
For mapping dynamic fields, Alert Details can be enriched. 
Source: https://learn.microsoft.com/en-us/azure/sentinel/customize-alert-details

I understand that this might not be exactly what you were looking for and thus happy to discuss this further.

Reply:
Hi @devikamehra, we have removed the Account Detection data mapping with AuditEvent Schema.